.changelog/4437.yml

```
@@ -0,0 +1,4 @@
+changes:
+- description: The ***unify*** command now logs the pack name and version in Python files.
```

```suggestion
- description: The ***unify*** command now logs the pack name and version in Python files.
```

doesn't it happen (also?) in `prepare-content`?

---

I made changes to the "unify" command, probably in the "prepare-content" we are calling to the "unify" command.

---

I think it's worth mentioning both